trustpoint/est/views.py

```diff
@@ -55,8 +58,11 @@ def __init__(self, content: str | bytes = b'', status: int | None = None, *args:
         if status and status >= THRESHOLD_LOGGER:
             if isinstance(content, bytes):
                 content = content.decode('utf-8')
-            error_message = f'EST Error - {status} - {content}'
+            error_message = f'EST - ERROR - {status} - {content}'
             self.logger.error(error_message)
+        else:
+            success_message = f'EST - SUCCESS - {status}'
```

Comment:
Tiny thing, Alex and I determined yesterday that doing `self.logger.info('EST - SUCCESS - %s', status)` makes most sense for inserting vars into logging strings